atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/TracingKeyValueService.java

```diff
@@ -244,32 +251,20 @@ public void dropTables(Set<TableReference> tableRefs) {
     public ClosableIterator<RowResult<Value>> getRange(TableReference tableRef,
             RangeRequest rangeRequest,
             long timestamp) {
-        //noinspection unused - try-with-resources closes trace
-        try (CloseableTrace trace = startLocalTrace("getRange({}, ts {})",
-                tableRef, timestamp)) {
-            return delegate().getRange(tableRef, rangeRequest, timestamp);
-        }
+        return delegate().getRange(tableRef, rangeRequest, timestamp);
     }
 
     @Override
     public ClosableIterator<RowResult<Set<Long>>> getRangeOfTimestamps(TableReference tableRef,
             RangeRequest rangeRequest,
             long timestamp) {
-        //noinspection unused - try-with-resources closes trace
-        try (CloseableTrace trace = startLocalTrace("getRangeOfTimestamps({}, ts {})",
-                tableRef, timestamp)) {
-            return delegate().getRangeOfTimestamps(tableRef, rangeRequest, timestamp);
-        }
+        return delegate().getRangeOfTimestamps(tableRef, rangeRequest, timestamp);
     }
 
     @Override
     public ClosableIterator<List<CandidateCellForSweeping>> getCandidateCellsForSweeping(TableReference tableRef,
             CandidateCellForSweepingRequest request) {
-        //noinspection unused - try-with-resources closes trace
-        try (CloseableTrace trace = startLocalTrace("getCandidateCellsForSweeping({}, ts {})",
-                tableRef, request.maxTimestampExclusive())) {
-            return delegate().getCandidateCellsForSweeping(tableRef, request);
-        }
+        return delegate().getCandidateCellsForSweeping(tableRef, request);
```

Comment:
Probably worth a comment here why we’re not tracing due to span not capturing full unit of work due to lazy iterator